File: repos/Lab3CSharp/Lab3CSharp/Question2.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Lab3CSharp
{
    class Question2
    {
        public void qTwo() 
        {

            Console.Write("Number of words to build: ");
            int num = Convert.ToInt32(Console.ReadLine());
            string[] words = new string[num];
            for (int i = 0; i < num; i++)
            {
                Console.Write("Enter the name " + (i + 1) + ": ");
                words[i] = Console.ReadLine();
            }

            Console.Write("World builder result is ");
            for (int i = 0; i < words.Length - 1; i++)
            {
                Console.Write(words[i] + ", ");
            }
            
            Console.Write(words[words.Length - 1]);
            Console.WriteLine("\nDONE!!");
        }

    }
}

File: Java/AdvProg/Lab3/src/questions/Question2.java
package questions;
import java.util.Scanner;
import java.util.Arrays;

public class Question2 {

	public static void main(String[] args) {
		Scanner input = new Scanner(System.in);
		
		System.out.print("Number of words to build: ");
		int num = input.nextInt();
		String[] words = new String[num];

		
		for (int i = 1; i <= num; i++) {
			
			System.out.print("Enter word " + i + ": ");
			String word = input.next();
			words[i - 1] = word; 
			
		}
			
		System.out.print("Word Builder Result = ");
		for (int i = 0; i < num; i++) {
			System.out.print(words[i] + " ");
		}
		System.out.println("\nDONE!!");
		input.close();
	}
}
